inventory.pvfs

```diff
@@ -0,0 +1,21 @@
+# Ansible Shade uses OpenStack clients running locally
```

Comment:
i had SSH config point to the nodes and sort out the keys. So ansible will be resolving names that way. not sure how this works with heat.